parsons/constants.py

```diff
@@ -1,7 +1,6 @@
 PARSONS_FOLDER_NAME = 'parsons_probs'
 PARSONS_FOLDER_PATH = './parsons_probs'
 PARSONS_GLOB = 'parsons_probs/*.py'
-PROBLEM_PATHS = [PARSONS_FOLDER_PATH]
```

Comment:
Given this is always a list with one constant, the added complexity of its existence doesn't seem warranted.